src/flb_output.c

```diff
@@ -452,6 +537,13 @@ struct flb_output_instance *flb_output_new(struct flb_config *config,
     instance->is_threaded = FLB_FALSE;
     instance->tp_workers = plugin->workers;
 
+    instance->singleplex_queue = flb_task_queue_create();
```

Comment:
need to delete

Reply:
queue should only be created if we are in singleplex mode?

Reply:
I added this comment reminding myself to do the memory clean up when the out_ins is deleted

Reply:
Pending